recipe/conda_build_config.yaml

```diff
@@ -200,12 +206,12 @@ vc:
   - 14
   - 14
 vlfeat:
-  - 0.9
+  - 0.9.20
 xerces_c:
-  - 3
+  - 3.2.0
 xz:
-  - 5
+  - 5.2
 zeromq:
-  - 4.2
+  - 4.2.1  # should be 4.2.*, but ABI is incorrectly tagged
 zlib:
-  - 1.2
+  - 1.2.11
```

Comment:
Will being more specific here break anything?

Reply:
This is the [current `zlib` pinning in the webpage repo]( https://github.com/conda-forge/conda-forge.github.io/blob/61cdd7fd0374c07e95afdcbd414c7985b141e4b3/scripts/pin_the_slow_way.py#L129 ). So no it shouldn't.

Some packages were implicitly using this version when they had the `zlib 1.2.*` pinning. In those cases, this pinning should fix breakage. Though ultimately those packages need to be rebuilt with a tighter pinning.

Also please see [my old comment]( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/7#discussion_r169865441 ) for details about ABI breakage between patch versions, which motivates this tight pinning.